interfaces/cython/SConscript

```diff
@@ -89,96 +89,72 @@ elif localenv['OS'] == 'Cygwin':
 localenv.Depends(ext, localenv['cantera_staticlib'])
 
 for f in (multi_glob(localenv, 'cantera', 'py') +
-          multi_glob(localenv, 'cantera/test', 'py') +
-          multi_glob(localenv, 'cantera/examples/kinetics', 'py') +
-          multi_glob(localenv, 'cantera/examples/multiphase', 'py') +
-          multi_glob(localenv, 'cantera/examples/onedim', 'py') +
-          multi_glob(localenv, 'cantera/examples/reactors', 'py') +
-          multi_glob(localenv, 'cantera/examples/surface_chemistry', 'py') +
-          multi_glob(localenv, 'cantera/examples/thermo', 'py') +
-          multi_glob(localenv, 'cantera/examples/transport', 'py')):
+          multi_glob(localenv, 'cantera/*', 'py') +
+          multi_glob(localenv, 'cantera/*/*', 'py')):
     localenv.Depends(mod, f)
 
 # Determine installation path and install the Python module
+install_cmd = ["$python_cmd_esc", "-m", "pip", "install"]
+user_install = False
+python_prefix = None
 if localenv['python_prefix'] == 'USER':
     # Install to the OS-dependent user site-packages directory
-    extra = '--user'
-    if localenv['OS'] == 'Darwin':
-        extra += ' --prefix=""'
-elif localenv['python_prefix']:
+    install_cmd.append("--user")
+    user_install = True
+elif localenv["python_prefix"]:
     # A specific location for the Cantera python module has been given
-    if localenv['debian'] and localenv.subst('${python_prefix}') == '/usr/local':
-        # Installation to /usr/local is the default on Debian-based distributions
-        extra = ''
-    elif localenv['OS'] == 'Darwin':
-        extra = localenv.subst(' --prefix=${python_prefix}')
-    elif localenv['libdirname'] != 'lib':
-        # 64-bit RHEL / Fedora etc. or e.g. x32 Gentoo profile
-        extra = localenv.subst(
-            ' --prefix=${{python_prefix}} --install-scripts=${{python_prefix}}/bin'
-            ' --install-lib=${{python_prefix}}/${{libdirname}}/python{}/site-packages'.format(py_version_short))
-    else:
-        extra = '--user'
-        localenv.AppendENVPath(
-            'PYTHONUSERBASE',
-            normpath(localenv.subst('$python_prefix')))
-else:
-    # Install Python module in the default location
-    extra = ''
+    install_cmd.append(f"--prefix={localenv.subst('python_prefix')}")
+    python_prefix = localenv.subst("python_prefix")
 
 if env["stage_dir"]:
     # Get the absolute path to the stage directory. If the stage directory is a relative
-    # path, consider it to be relative to the root of the Cantera source directory,
-    # which is two directories up from the current working directory of this SConscript
-    # file.
-    stage_absolute = Path.cwd().parents[1].joinpath(env['stage_dir']).resolve()
-    extra += f" --root={stage_absolute}"
+    # path, consider it to be relative to the root of the Cantera source directory.
+    stage_dir = Path(env["stage_dir"])
+    if not stage_dir.is_absolute():
+        stage_dir = Path(Dir("#").abspath) / stage_dir
```

Comment:
I think this is more robust than `Path.cwd().parents[1]` if the stage directory is relative, and that I haven't broken the fixes from #1102, but I don't have a FreeBSD machine to test. @yurivict can you confirm this works for you?

Reply:
Fwiw, this is the exact line that gave me grief in #1191.

Reply:
Were you working off of this branch? This line changed in this PR

Reply:
No. But my troubles started at the line this PR is changing.